Bottom: 0378c1b75d744b930aa371128cb7b25216b64630
Top:    c0d855b06e5344a620928af2cbef95caaa745d65
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-01-27 22:22:43 +0000

Fix for PCB+GL Experimental


---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 6ad5917..feb9522 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -852,9 +852,9 @@ do_hole (const BoxType *b, void *cl)
 }
 
 static bool
-polygon_contains_user_holes (PolygonType *polygon)
+pour_contains_user_holes (PourType *pour)
 {
-  return (polygon->HoleIndexN > 0);
+  return (pour->HoleIndexN > 0);
 }
 
 
@@ -888,7 +888,7 @@ hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box /*, bool forc
   /* Polygon has holes.. does it have any user-drawn holes?
    * If so, it must be masked with a _new_ stencil bit.
    */
-  use_new_stencil = force_new_stencil | polygon_contains_user_holes (poly);
+  use_new_stencil = force_new_stencil || true; // | polygon_contains_user_holes (poly);
 
   if (use_new_stencil)
     {
